Sources/PostgresNIO/Connection/PostgresConnection+Authenticate.swift

```diff
@@ -7,7 +7,7 @@ extension PostgresConnection {
         username: String,
         database: String? = nil,
         password: String? = nil,
-        logger: Logger = .init(label: "codes.vapor.postgres")
+        logger: Logger? = .init(label: "codes.vapor.postgres")
```

Comment:
Should this be `= nil`? 🤔

Reply:
Well, it's still nice to have it by default. My intent was never to remove the ability to see what's happening by default, and there are plenty of database messages you _definitely_ want to see in the console if they happen. My expectation for the optionality of it is to have an easy way to silence the logging in cases like unit tests where the `.trace` logging level is likely to be used by the top-level application and scrolling through reports of every last `PostgresMessage` on the connection is tedious 🙂. If either the default `LogHandler` or `console-kit`'s permitted filtering by category, I would use that, but adding that capability is _highly_ nontrivial and doesn't negate the separate uses for this change.